docs/intellij-automated-setup-behavior.md

```diff
@@ -357,7 +357,7 @@ The syntax for the pre-configured settings found below are as follows:
 | IntelliJ Setting        | Node                                              | Value                                                |
 |-------------------------|---------------------------------------------------|------------------------------------------------------|
 | `Configuration file:`   | `/custom-configuration-file[@used='true'][@path]` | `$PROJECT_DIR$/static-analysis/teammates-eslint.yml` |
-| `Extra eslint options:` | `/extra-options[@value]`                          | `--ext .es6 --ignore-pattern '**/*.js'`                                         |
+| `Extra eslint options:` | `/extra-options[@value]`                          | `--ignore-pattern 'node_modules/**/*' 'src/main/webapp/js/*.js' 'src/main/webapp/test/*.js'` |
```

Comment:
Can you change this to --ignore-pattern 'src/main/webapp/js/*.js' --ignore-pattern 'src/main/webapp/test/*.js' --ignore-pattern 'test-output/**/*.js'?